packages/jest-preset-mc-app/jest-preset.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
   setupFilesAfterEnv: [resolveRelativePath('./setup-test-framework.js')],
   snapshotSerializers: ['enzyme-to-json/serializer'],
   testEnvironment: 'jsdom',
```

Comment:
In theory this is a breaking change creeping in here. It will change tests (and does on the MC internally) and a user of the package has to change something. Making it crossing borders of being a breaking change 😺💀

Reply:
You mean the `testURL`?

Reply:
Yeah good catch, I’ll revert that. It does not matter much to use the new hostname for tests.

Reply:
We can also keep it now that the change is made but add it to the changelog as to how to migrate.